website/pages/api-docs/acl/auth-methods.mdx

```diff
@@ -54,7 +54,7 @@ The table below shows this endpoint's support for
 - `MaxTokenTTL` `(duration: 0s)` - This specifies the maximum life of any token
   created by this auth method. When set it will initialize the
   [`ExpirationTime`](/api/acl/tokens.html#expirationtime) field on all tokens
-  to a value of `Token.CreateTime + AuthMethod.MaxTokenTTL`.  This field is not
+  to a value of `Token.CreateTime + AuthMethod.MaxTokenTTL`. This field is not
```

Comment:
Luckily I finally figured out the cause of this was never setting `set nojoinspaces` in my vim config, so hopefully I won't keep inadvertently re-adding double-spaces.

Reply:
OH @rboyer I didn't know that was a thing, I am a common offender of double spacing after periods, 100% adding this to my vim config.